go.mod

```diff
@@ -184,5 +186,5 @@ require (
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.5.1-sdk-v0.46.6
```

Comment:
should we cut https://github.com/celestiaorg/cosmos-sdk v1.6.0-sdk-v0.46.7 and bump to that here given the diff above

```diff
-	github.com/cosmos/cosmos-sdk v0.46.6
+	github.com/cosmos/cosmos-sdk v0.46.7
```

Reply:
Already did. Just haven't updated it here. But will do in a separate PR